google/cloud/storage/internal/async/writer_connection_resumed_test.cc

```diff
@@ -217,9 +222,9 @@ TEST(WriteConnectionResumed, FlushNonEmpty) {
   MockFactory mock_factory;
   EXPECT_CALL(mock_factory, Call).Times(0);
 
-  auto connection =
-      MakeWriterConnectionResumed(mock_factory.AsStdFunction(), std::move(mock),
-                                  initial_request, nullptr, Options{});
+  auto connection = MakeWriterConnectionResumed(
```

Comment:
Can we please add a test case where we check if the generation is added to append_object_spec in case of error?